cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -1381,18 +1488,18 @@ public void testWithExternalRouteListenersWithSaslAuth()  {
     }
 
     @ParallelTest
-    public void testWithExternalRouteListenersWithCustomCerts()  {
+    public void testWithExternalRouteListenersWithCustomCerts() {
         GenericKafkaListener listener = new GenericKafkaListenerBuilder()
                 .withName("external")
                 .withPort(9094)
                 .withType(KafkaListenerType.ROUTE)
                 .withTls(true)
                 .withNewConfiguration()
-                    .withNewBrokerCertChainAndKey()
-                        .withSecretName("my-secret")
-                        .withKey("my.key")
-                        .withCertificate("my.crt")
-                    .endBrokerCertChainAndKey()
+                .withNewBrokerCertChainAndKey()
+                .withSecretName("my-secret")
+                .withKey("my.key")
+                .withCertificate("my.crt")
+                .endBrokerCertChainAndKey()
```

Comment:
I think the indentations made sense and should be kept. All calls can be chained but these are for nested classes

Reply:
Same below